<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLampHistoryDataDialog</class>
 <widget class="QDialog" name="CLampHistoryDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>路灯历史数据</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>开始时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_start">
       <property name="dateTime">
        <datetime>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2018</year>
         <month>1</month>
         <day>30</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>结束时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_end">
       <property name="dateTime">
        <datetime>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2018</year>
         <month>2</month>
         <day>3</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>气象信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_weather_info">
       <item>
        <property name="text">
         <string>温度(°C)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>湿度(%)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>风速(km/h)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PM10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PM2.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NO2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SO2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>O3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>光照</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>降水量(mm)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>噪声(dB)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>红外感应值</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_view">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="EchartsWebView" name="webView" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EchartsWebView</class>
   <extends>QWidget</extends>
   <header>third/echarts/echartswebview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
